plugins/woocommerce/src/Internal/FraudProtection/AdminSettingsHandler.php

```diff
@@ -0,0 +1,201 @@
+		// Only add on the features section.
+		if ( 'features' !== $current_section ) {
+			return $settings;
+		}
+
+		// Check if field already exists to prevent duplicates.
+		foreach ( $settings as $setting ) {
+			if ( isset( $setting['id'] ) && 'woocommerce_fraud_protection_jetpack_connection' === $setting['id'] ) {
+				return $settings;
+			}
+		}
+
+		// Find the fraud_protection field and add Jetpack connection field after it.
+		$new_settings = array();
+		foreach ( $settings as $setting ) {
+			$new_settings[] = $setting;
+
+			// Add Jetpack connection field after fraud_protection checkbox.
+			if ( isset( $setting['id'] ) && 'woocommerce_feature_fraud_protection_enabled' === $setting['id'] ) {
+				$new_settings[] = array(
+					'id'    => 'woocommerce_fraud_protection_jetpack_connection',
+					'type'  => 'jetpack_connection',
+					'title' => __( 'Jetpack Connection', 'woocommerce' ),
+					'desc'  => __( 'Connect your site to Jetpack to enable fraud protection features.', 'woocommerce' ),
+				);
+			}
+		}
+
+		return $new_settings;
+	}
+
+	/**
+	 * Output the Jetpack connection field.
+	 *
+	 * @internal
+	 *
+	 * @param array $value Field configuration.
+	 * @return void
+	 */
+	public function handle_output_jetpack_connection_field( $value ): void { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.Found
+		// Only show Jetpack connection when fraud protection is enabled.
+		if ( 'yes' !== get_option( 'woocommerce_feature_fraud_protection_enabled', 'no' ) ) {
+			return;
+		}
+
+		$this->output_jetpack_connection_status();
+	}
+
+	/**
+	 * Output the Jetpack connection status and button.
+	 *
+	 * @internal
+	 *
+	 * @return void
+	 */
+	private function output_jetpack_connection_status(): void {
+		// Get connection status from connection manager.
+		$connection_status = $this->connection_manager->get_connection_status();
+
+		?>
+		<tr valign="top">
+			<th scope="row" class="titledesc">
+				<label><?php esc_html_e( 'Jetpack Connection', 'woocommerce' ); ?></label>
+			</th>
+			<td class="forminp forminp-button">
+				<?php if ( ! $connection_status['connected'] ) : ?>
+					<?php
+					// Get authorization URL for connecting.
+					$redirect_url   = admin_url( 'admin.php?page=wc-settings&tab=advanced&section=features' );
+					$connection_url = $this->connection_manager->get_authorization_url( $redirect_url );
+
+					// If we couldn't get authorization URL, show error message.
+					if ( ! $connection_url ) :
+						?>
+						<p class="description" style="color: #dc3232;">
+							<?php echo esc_html( $connection_status['error'] ); ?>
+						</p>
+					<?php else : ?>
+						<button
+							type="button"
+							class="button button-secondary jetpack_connection_button"
+							data-connection-url="<?php echo esc_url( $connection_url ); ?>"
+						>
+							<?php esc_html_e( 'Connect to Jetpack', 'woocommerce' ); ?>
+						</button>
```

Comment:
Could the connect button could be a simple link?

```php
<a href="<?php echo esc_url( $connection_url ); ?>" class="button button-secondary jetpack_connection_button">
    <?php esc_html_e( 'Connect to Jetpack', 'woocommerce' ); ?>
</a>
```

This would remove the need for the inline JavaScript and jQuery dependency.